src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -20,31 +20,29 @@
 */
 
 #include <yuni/yuni.h>
-#include "antares/solver/optimisation/opt_structure_probleme_a_resoudre.h"
 
-#include "antares/solver/simulation/simulation.h"
-#include "antares/solver/simulation/sim_structure_probleme_economique.h"
 #include "antares/solver/optimisation/opt_fonctions.h"
+#include "antares/solver/optimisation/opt_structure_probleme_a_resoudre.h"
+#include "antares/solver/simulation/sim_structure_probleme_economique.h"
+#include "antares/solver/simulation/simulation.h"
 
 extern "C"
 {
 #include "spx_definition_arguments.h"
 #include "spx_fonctions.h"
-
 #include "srs_api.h"
 }
 
-#include <antares/logs/logs.h>
-#include <antares/antares/fatal-error.h>
+#include <chrono>
```

Comment:
STL inclusion first ?

Reply:
Yes, here we need to do something by hand because of the "extern C" block above. Clang-format will not remove it alone.